### PR TITLE
[MB-14368] Update unsupported document viewer view and add viewer to Review Documents page

### PR DESCRIPTION
**Pull request by pearl-truss**
## Summary

This PR does a few things, it:

- Displays the changes introduced in `react-file-viewer` to display the updated UI for unsupported files, for XLS/XLXS files
- Adds DocumentViewer to the ReviewDocuments page
- Added a throughline to fetch weightTickets 

NOTE: 
- changes are pending to resolve the server test 
- There are follow up tickets for fully integrating the document viewer on the page

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Follow the [S3 instructions](https://dp3.atlassian.net/wiki/spaces/MT/pages/1470955567/How+to+test+storing+data+in+S3+locally). Before step 3, run `make clean` and then `make client_run`
2. Login to customer app as the `readyToFinish@ppm.approved` customer. Note the move code
3. Click `Upload PPM Documents` and upload files 
4. Sign into the Office app as `too_tio_services_counselor_role@office.mil`
5. As a TOO find the move of the `readyToFinish@ppm.approved` customer. Click 'Edit shipment', click through app. Back to the page, click 'Review documents'
6. You should see the document viewer working on this page

STORYBOOk
There is an XLS example story added to storybook under the DocumentViewer Content story

## Screenshots
Screen Shot 2022-12-22 at 11 12 10 AM
Screen Shot 2022-12-22 at 12 21 42 PM
Screen Shot 2022-12-22 at 12 21 53 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
-    "@trussworks/react-file-viewer": "git+https://github.com/trussworks/react-file-viewer",
+    "@trussworks/react-file-viewer": "git+https://github.com/trussworks/react-file-viewer#pm-add-xls-unsupported-view",
```
Comment (pearl-truss) [inline package.json:9]: **NOTE:** the react-file-viewer PR will need to be merged before this one. And once that other PR is merged, this should be updated to remove the branch hash

Comment (NamibiaTorres) [reply package.json:9]: Thanks for noting this. Do you happen to have a link to this PR?

Comment (pearl-truss) [reply package.json:9]: It's here: https://github.com/trussworks/react-file-viewer 

Also, I'm closing this PR and will open another with the same changes, just a different branch (something got weird with this branch with git)

File: pkg/uploader/allowed_file_types.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ type AllowedFileTypes []string
 
 var (
 	// AllowedTypesServiceMember are the content types we allow service members to upload for orders
-	AllowedTypesServiceMember AllowedFileTypes = []string{"image/jpeg", "image/png", "application/pdf"}
+	AllowedTypesServiceMember AllowedFileTypes = []string{"image/jpeg", "image/png", "application/pdf", "application/vnd.ms-excel", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"}
```
Comment (NamibiaTorres) [inline pkg/uploader/allowed_file_types.go:8]: **Question**: Looks like we are expanding file types? I'm not too familiar with .sheet, that is a type of xml?

File: package.json (hunk 1 above)
Comment (NamibiaTorres) [reply package.json:9]: Ok, no problem! I'll just pick up where I left off with the review.

**Pull request closed without merging**